app/controllers/application_controller.rb

```diff
@@ -47,6 +47,13 @@ def access_denied
     render file: 'public/403.html', status: 403
   end
 
+  def individual_market_is_active
+    if !(Settings.aca.individual_market.is_active)
```

Comment:
This is fine for now - but we want to use the actual array of market_kinds and do a include? not add another key in the settings